### ISSUE TITLE
Widespread warning messages about BadJetIDVersion in IB RelVals

### ISSUE THREAD

**Issue by mmusich** [cms-sw/cmssw#31171]
Basically in all step3's of the IB matrix there are warnings of the type:
```
%MSG-w BadJetIDVersion:  JetAnalyzer:jetDQMAnalyzerAk4PFUncleaned@ctor  19-Aug-2020 07:33:03 CEST pre-events
The LOOSE operating point is only supported for the WINTER16 JetID version -- defaulting to TIGHT
%MSG
```

see e.g: [Run1 data](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-18-2300/pyRelValMatrixLogs/run/4.17_RunMinBias2011A+RunMinBias2011A+HLTD+RECODR1reHLT+HARVESTDR1reHLT+SKIMDreHLT/step3_RunMinBias2011A+RunMinBias2011A+HLTD+RECODR1reHLT+HARVESTDR1reHLT+SKIMDreHLT.log),  [Run2 data](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-18-2300/pyRelValMatrixLogs/run/4.55_RunJet2012B+RunJet2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT/step3_RunJet2012B+RunJet2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT.log), [Run2 MC](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-18-2300/pyRelValMatrixLogs/run/1311.0_MinBias_13+MinBias_13INPUT+DIGIUP15+RECOMINUP15+HARVESTMINUP15+ALCAMINUP15/step3_MinBias_13+MinBias_13INPUT+DIGIUP15+RECOMINUP15+HARVESTMINUP15+ALCAMINUP15.log), [Run3 MC](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-18-2300/pyRelValMatrixLogs/run/11650.0_ZMM_14+2021+ZMM_14TeV_TuneCP5_GenSimINPUT+Digi+Reco+HARVEST+ALCA/step3_ZMM_14+2021+ZMM_14TeV_TuneCP5_GenSimINPUT+Digi+Reco+HARVEST+ALCA.log), [Phase-2 MC](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_2_X_2020-08-18-2300/pyRelValMatrixLogs/run/30234.0_TTbar_14TeV+2026D65+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal/step3_TTbar_14TeV+2026D65+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal.log)

**Comment by slava77** [comment]
assign dqm

@ahinzmann  @camclean @alefisico 
since the warnings are coming from the constructor, I'm guessing this can be blamed on the configuration (not on the reco inputs to the module).
Is some era/process modifier missing to select the right configuration?

**Comment by ahinzmann** [comment]
Is this a new feature?
Does this appear already before #31125 is merged?

**Comment by ahinzmann** [comment]
it is probably due to 
https://github.com/cms-sw/cmssw/pull/30638/files#r473007749

**Comment by ahinzmann** [comment]
Fix is #31177

**Comment by mmusich** [comment]
closing, fixed by #31177
